spec/system/provider_interface/see_applications_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'rails_helper'
+
+RSpec.feature 'See applications' do
+  scenario 'Provider visits application page' do
+    given_i_am_a_provider_user
+    and_my_organisation_has_new_applications
+    and_i_visit_the_provider_page
+    then_i_should_see_the_applications_from_my_organisation
+    but_not_the_applications_from_other_providers
+
+    when_i_click_on_an_application
+    then_i_should_be_on_the_application_view_page
+  end
+
+  def given_i_am_a_provider_user
+    # This is stubbed out for now in the controller.
+  end
+
+  def and_my_organisation_has_new_applications
+    create(:application_choice, status: 'application_complete', provider_ucas_code: 'ABC', application_form: create(:application_form, first_name: 'Alice', last_name: 'Wunder'))
```

Comment:
Since we're setting `status`es, is it worth adding an assertion about not seeing an application that's not `complete`?

Reply:
Removed the status for now in https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/272/commits/efb4923fb81bfb7a104ed8e6d47d9d501aca36dd, we'll revisit that later!